docs/pages/validator-management/vc-configuration.md

```diff
@@ -17,7 +17,7 @@ The mnemonic is randomly generated during wallet creation and printed out to the
 
 ### Create a wallet
 
-Lodestar is deprecating its functionality to create wallets.
+Lodestar has removed its functionality to create wallets.
 
 To create a wallet, we recommend using the official [`staking-deposit-cli`](https://github.com/ethereum/staking-deposit-cli/releases) from the Ethereum Foundation for users comfortable with command line interfaces.
```

Comment:
The term wallet here might be a bit outdated, maybe should use term keystores, or we use same wording as [deposit cli introduction](https://github.com/ethereum/staking-deposit-cli?tab=readme-ov-file#introduction).

Maybe "keystores and corresponding deposit data" but since the whole section is about "Create a wallet" maybe better to revise that part in a separate PR.

Reply:
Will follow up in another PR with the best way to align this. TBH, I'm still kinda wondering if we need this as generally, most go through the Ethereum launchpad which provides all these instructions for them already. This whole section has been around for a long time when Lodestar had these features and before the existence of a lot of this tooling + other references.